sympy/matrices/dense.py

```diff
@@ -1281,16 +1281,16 @@ def hessian(f, varlist, constraints=[]):
     return out
 
 
-def jordan_cell(eigenval, n):
+def jordan_block(eigenval, n):
```

Comment:
I did not find `from matrices.dense import jordan_cell` in dense.py . I am stuck at this.Need your help @siefkenj .